src/test/java/org/opensearch/knn/plugin/script/PainlessScriptIT.java

```diff
@@ -126,22 +128,42 @@ private List<MappingProperty> buildMappingProperties() {
 
     public void testL2ScriptScoreFails() throws Exception {
         String source = String.format("1/(1 + l2Squared([1.0f, 1.0f], doc['%s']))", FIELD_NAME);
-        Request request = buildPainlessScriptRequest(source, 3, getL2TestData());
+        Request request = buildPainlessScoreScriptRequest(source, 3, getL2TestData());
```

Comment:
Is it PainlessScriptScore or PainlessScoreScript ? What is the actual naming convention ?

Reply:
PainlessScripting -> to represent we want to test using painless script examples.
Score/ScriptedMetrics -> contexts

Reply:
Will rename accordingly